evap/staff/tests/test_views.py

```diff
@@ -886,6 +902,40 @@ def test_remove_responsibility(self):
 
         self.assertIn("No responsible contributors found", page)
 
+    def test_remove_participant(self):
```

Comment:
Maybe rename to "test_participant_removal_reward_point_granting_message"?
Maybe let mommy set the username and compare to the (random) name in the assertion?